site/dat/docs/CommandLine.md

```diff
@@ -1,4 +1,4 @@
-# Command-Line Tool
+# Command-Line Tools
```

Comment:
Nope, this is not the right place to document the converter. Create a dedicated doc page for it.